src/lib/Libnet/net_server.c

```diff
@@ -1044,11 +1001,10 @@ init_poll_context(void)
 		max_connection = nfiles;
 
 #endif
-	DBPRT(("#init_poll_context: initing poll_context for %d\n",max_connection))
+	DBPRT(("#init_poll_context: initializing poll_context for %d", max_connection))
 	poll_context = tpp_em_init(max_connection);
 	if (poll_context == NULL) {
-		snprintf(logbuf, sizeof(logbuf), "Could not init poll_context");
-		log_err(-1, __func__, logbuf);
+		log_err(errno, __func__, "could not initialize poll_context");
```

Comment:
Why change the text?

Reply:
Just making it nicer...prefer to revert back?

Reply:
If the tests pass, I'm fine with the change. I'm simply concerned about regressions. These are the kind of "common sense" changes that should not require an EDD, IMHO.